handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,264 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.
+
+DataPlugins can be created by using C++, VBS, LabVIEW or Python.

```

Comment:
To make this more direct/active: 'You can create DataPlugins using C++, VBS...'

Reply:
thanks, I changed it.